### ISSUE TITLE
Get `AST is missing the tokens array.`!

### ISSUE THREAD

**Issue by thearabbit** [mysticatea/vue-eslint-parser#18]
Get `AST is missing the tokens array.`!
```js
// eslint
{
    "extends": [
        "eslint:recommended",
        "plugin:vue/recommended" // or "plugin:vue/base"
    ],
    "parser": "vue-eslint-parser",
    "parserOptions": {
        "parser": "babel-eslint",
        "ecmaVersion": 2017,
        "sourceType": "module"
    },
    "rules": {
        "no-console": ["error"],
        "semi": ["error", "always"],
        "quotes": ["error", "single"]
        // override/add rules" settings here
        // "vue/valid-v-if": "error"
    }
}
-------------
// package
"devDependencies": {
    "@types/lodash": "^4.14.74",
    "babel-eslint": "^7.2.3",
    "babel-helper-vue-jsx-merge-props": "^2.0.2",
    "babel-plugin-syntax-jsx": "^6.18.0",
    "babel-plugin-transform-vue-jsx": "^3.5.0",
    "babel-preset-es2015": "^6.24.1",
    "eslint": "^4.6.1",
    "eslint-plugin-vue": "^3.13.0",
    "vue-eslint-parser": "^1.0.0"
  }
```

**Comment by mysticatea** [comment]
Thank you for this report.

I will check it later. I'm sorry that I can't do in this holiday because I'm in traveling.

**Comment by mysticatea** [comment]
I apology for late response.

After re-look your config, I noticed that you have `"vue-eslint-parser": "^1.0.0"` as a dependency.
`eslint-plugin-vue@beta` needs `vue-eslint-parser@^2.0.1-beta.0`.

Could you confirm to work after removing the following two?:

- `"parser": "vue-eslint-parser",` in .eslintrc
- `"vue-eslint-parser": "^1.0.0"` in package.json (and uninstall it)

**Comment by thearabbit** [comment]
Don't show this error, after do this.

**Comment by mysticatea** [comment]
Closing as resolved.
